AllenC9/NAROS
# Revert "Create useless.c"

## Description


## Motivation and Context


## Issue Link
https://ekasolutions.atlassian.net/browse/EB-1807

## How Do I Test This?

1. when we click truck or trailer name we should get this modal:
were we can see all the asset assignments and loads that have been assigned to the assets
![Screen Shot 2020-04-10 at 4 38 35 PM](https://user-images.githubusercontent.com/28992212/79024838-be138200-7b49-11ea-90d0-0dece99bd93c.png)

2. when clicking a driver name we should see similar to the modal for truck and trailer but first we should see some inputs to update the info of the carrier:
![Screen Shot 2020-04-10 at 4 39 50 PM](https://user-images.githubusercontent.com/28992212/79024895-e9966c80-7b49-11ea-9029-50ff3fe20e1e.png)

3. update the add and edti asset assigment modal:
edit new view:
![Screen Shot 2020-04-10 at 4 40 37 PM](https://user-images.githubusercontent.com/28992212/79024961-0e8adf80-7b4a-11ea-8e56-701c3712a208.png)

add new asset assignment:
![Screen Shot 2020-04-10 at 4 41 41 PM](https://user-images.githubusercontent.com/28992212/79024998-29f5ea80-7b4a-11ea-86bb-5c35a0b279a8.png)

4. loads assigned preview:
![Screen Shot 2020-04-10 at 4 41 56 PM](https://user-images.githubusercontent.com/28992212/79025019-3a0dca00-7b4a-11ea-8015-1d9e13529a57.png)

## Checklist:
